source/sass/blocks/form.scss

```
@@ -1,18 +1,49 @@
 .form fieldset {
   margin: 0;
   padding: 0;
+  padding-top: 18px;
   padding-left: 20px;
   padding-right: 20px;
```

Лучше задавать свойства явно или использовать сокращенное правило
```suggestion
  padding-bottom: 0;
  padding-top: 18px;
  padding-left: 20px;
  padding-right: 20px;
```

```suggestion
  padding: 18px 20px 0 20px;
```